6/ansible-6.0.0b1.deps

```diff
@@ -0,0 +1,103 @@
+_ansible_version: 6.0.0b1
+_ansible_core_version: 2.13.0
+amazon.aws: 3.2.0
+ansible.netcommon: 3.0.0
+ansible.posix: 1.4.0
+ansible.utils: 2.6.1
+ansible.windows: 1.10.0
+arista.eos: 5.0.0
+awx.awx: 21.0.0
+azure.azcollection: 1.12.0
+check_point.mgmt: 2.3.0
+chocolatey.chocolatey: 1.2.0
+cisco.aci: 2.2.0
+cisco.asa: 3.0.0
+cisco.dnac: 6.4.0
+cisco.intersight: 1.0.18
+cisco.ios: 3.0.0
+cisco.iosxr: 3.0.0
+cisco.ise: 2.4.1
+cisco.meraki: 2.6.2
+cisco.mso: 2.0.0
+cisco.nso: 1.0.3
+cisco.nxos: 3.0.0
+cisco.ucs: 1.8.0
+cloud.common: 2.1.1
+cloudscale_ch.cloud: 2.2.2
+community.aws: 3.2.1
+community.azure: 1.1.0
+community.ciscosmb: 1.0.5
+community.crypto: 2.3.1
+community.digitalocean: 1.19.0
+community.dns: 2.1.1
+community.docker: 2.6.0
+community.fortios: 1.0.0
+community.general: 5.0.0
+community.google: 1.0.0
+community.grafana: 1.4.0
+community.hashi_vault: 3.0.0
+community.hrobot: 1.3.0
+community.libvirt: 1.1.0
+community.mongodb: 1.4.0
+community.mysql: 3.2.1
+community.network: 3.3.0
+community.okd: 2.2.0
+community.postgresql: 2.1.5
+community.proxysql: 1.4.0
+community.rabbitmq: 1.2.1
+community.routeros: 2.1.0
+community.sap: 1.0.0
+community.sap_libs: 1.1.0
+community.skydive: 1.0.0
+community.sops: 1.2.1
+community.vmware: 2.5.0
+community.windows: 1.10.0
+community.zabbix: 1.7.0
+containers.podman: 1.9.3
+cyberark.conjur: 1.1.0
+cyberark.pas: 1.0.14
+dellemc.enterprise_sonic: 1.1.0
+dellemc.openmanage: 5.3.0
+dellemc.os10: 1.1.1
+dellemc.os6: 1.0.7
+dellemc.os9: 1.0.4
+f5networks.f5_modules: 1.16.0
+fortinet.fortimanager: 2.1.5
+fortinet.fortios: 2.1.4
```

Comment:
Version 2.1.6 is available since a couple of hours. I'm not sure how exactly 2.1.5 is broken, but maybe this new version is OK now?

Reply:
2.1.5 had syntax errors in some of its Py files. That should be fixed in 2.1.6...